File: 2185-counting-words-with-a-given-prefix/2185-counting-words-with-a-given-prefix.py
class Solution:
    def prefixCount(self, words: List[str], pref: str) -> int:
        p, ans = len(pref), 0

        for word in words:
            if len(word) < p:
                continue
            j = 0
            for i in range(p):
                if word[i] == pref[i]:
                    j += 1
            if j == p:
                ans += 1
        
        return ans

File: 2185-counting-words-with-a-given-prefix/2185-counting-words-with-a-given-prefix.cpp
class Solution {
public:
    int prefixCount(vector<string>& words, string pref) {
        int count = 0;

        for (const string& word : words){
            if (word.find(pref) == 0)
                ++count;    // Found a string with pref as prefix
        }

        return count;
    }
};